Tests/SparkConnectTests/DataFrameTests.swift

```diff
@@ -81,6 +81,7 @@ struct DataFrameTests {
     await spark.stop()
   }
 
+#if !os(Linux)
```

Comment:
This is ignored on `Linux` environment due to the `Swift` library difference. Will be recovered later.

Reply:
Is it only for this test?

Reply:
Yes, only `show` related `3` tests.